slack-rtm.c

```diff
@@ -187,8 +187,15 @@ static gboolean rtm_connect_cb(SlackAccount *sa, gpointer data, json_value *json
 	slack_blist_init(sa);
 
 	slack_login_step(sa);
+
+	size_t cookie_size = strlen("d=") + strlen(sa->d_cookie) + 1;
```

Comment:
Got a segfault with today's update. This fixes it:

```suggestion
	size_t cookie_size = strlen("d=") + (sa->d_cookie ? strlen(sa->d_cookie) : 0) + 1;
```

Reply:
I can confirm that Pidgin without that suggestion crashes after starting Pidgin. The suggestion keeps Pidgin running.